extensions/ql-vscode/test/vscode-tests/cli-integration/jest.setup.ts

```diff
@@ -78,6 +79,15 @@ beforeAll(async () => {
   await extensions.getExtension("GitHub.vscode-codeql")?.activate();
 });
 
+beforeEach(async () => {
+  await beforeEachAction();
+
+  await CUSTOM_CODEQL_PATH_SETTING.updateValue(
+    process.env.CLI_PATH,
+    ConfigurationTarget.Global,
```

Comment:
I don't think the tests should be messing with global settings here.

```suggestion
    ConfigurationTarget.Workspace,
```

Reply:
👍, this shouldn't actually change any configuration settings for the user because the `beforeEachAction` has already run at this point which mocks the configuration. However, I do agree that if this isn't mocked it should update the workspace configuration.